### DISCUSSION TITLE
1.63.0 - слетели роуты login, logout

### DISCUSSION THREAD

**Discussion by torkvi-po** [moonshine-software/moonshine#557] [Q&A]
Доброго дня. Вчера обновил moonshine и слетели route name: login и logout.
Вместо основных breeze'овских начали отрабатывать moonshine'овские.
Глянул по коду, что они и раньше так же назывались, но отрабатыли правильно.
Пока руками изменил, но хотелось бы понять почему раньше работало и как жить дальше?))
Спасибо!

**Comment by lee-to** [comment]
Возможно у вас админка без префикса?
Тогда и муншайн будет иметь роут /login /logout и также и бриз, соответственно объявляться будет только первый
Если нет префикса то вам нужно с этим самостоятельно что то решать и дело не в 1.63

**Reply by torkvi-po** [reply]
с префиксом! всё норм! я про именованные пути:
вот auth.php от breeze:
    Route::get('login', [AuthenticatedSessionController::class, 'create'])
                **->name('_login_');**
вот routes/moonshine.php из пакета:
Route::get('/login', 'login')->**name('_login_')**;

А в коде breeze идёт, что если неправильные данные для входа, то redirect()->route('login').
И до этого он выбирал автоматом breeze, а сейчас начал moonshine

**Comment by lee-to** [comment]
все роуты муншайн названия с префиксом https://github.com/moonshine-software/moonshine/blob/1.5.x/routes/moonshine.php

```php
Route::prefix(config('moonshine.route.prefix', ''))
    ->middleware($middlewares)
    ->as('moonshine.')->group(static function () {
```

  GET|HEAD        admin/login ............................... moonshine.login › MoonShine\Http › AuthenticateController@login
